coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -212,8 +212,8 @@ def __init__(self):
             logger.info('Will not attempt koji write operations')
 
     def __call__(self, message: fedora_messaging.api.Message):
-        logger.debug(message.topic)
-        logger.debug(message.body)
+        #logger.debug(message.topic)
+        #logger.debug(message.body)
```

Comment:
Is there a reason why these lines aren't deleted rather than commented out?

Reply:
mostly so they'd be easy to add back without having to remember exactly the syntax (i.e. var name) again. I could be swayed to delete it but have a preference for leaving it in and commented.

Reply:
I can go either way with this also as long as there isn't commented code everywhere which is not the case for this.